status-updates/2023/2023-06-09.md

```diff
@@ -0,0 +1,233 @@
+
+RD: I'll work on announcements for our [upcoming events](#upcoming-events).
+Toe, I'll ask you to review them.
+We are targeting technical coordinators of various lab teams.
+They can give valuable feedback and also carry our insights over to their teams.
+
+RD: Sergiu is on holiday until next Tuesday.
+After that, he'll start working on the infrastructure paper afterwards.
+He was having trouble finding and getting accustomed to the state of the art as education is a new domain for him.
+
+TD: No updates on the OS content.
+The exams are taking place now and everyone is busy learning / grading.
+
+GM: I've been looking into the builder to see how it may fit with our needs.
+I also think we should use the landing page (which is currently [empty on the OS class](https://open-education-hub.github.io/operating-systems/)) to host administrative information.
+
+AA: Nothing on statistics.
+
+SJ: Talked to Eggert + Liza about quizzes.
+
+RD: Talk to Elena about the OS and IOS (Introduction to Operating Systems) quizzes.
+
+TD: I can also help with the OS (and HSI quizzes) because I've given them some thoughts.
+
+RD: Also keep the Networking quizzes in mind.
+
+MB: I will discuss the VMChecker-Next workshop with Sergiu.
+
+VN: I am improving the builder's documentation.
+
+LB: I have written down some ideas about future social media posts, but I haven't had the time to implement them.
+
+RD: Try to spread the word about the 27K assignments that have been run on VMChecker-Next, that's an impressive number.
+
+### OS 2023-2024 Organisation
+
+RD: have the team assembled and the calendar planned by July 1.
+
+RD: From next year, there will be no more undergraduate TAs.
+Therefore, the remaining TAs will have to teach multiple labs.
+If need be, we will get some more TAs.
+
+RD: What do you think about assigning subteams not per component like we're doing now (lectures, labs, assignments, etc.), but per chapter.
+This way, the lectures, labs and assignment that correspond to each chapter will be more in sync.
+This is also how we designed the new OS course.
+
+EM: There will be mismatches between different lectures, deployments and especially communication with students.
+I've had cases where someone would tell students one thing for a lecture quiz, while someone else would say it differently for another quiz, thus creating confusion.
+
+RD: Let's have another person dedicated to deployment.
+For example, this year I took care of deploying assignments and it went well.
+By deployment I mean creating the assignment on Gitlab for students to make PRs, uploading quizzes on Moodle etc.
+
+TD: Why?
+Deployment is usually something that can easily be described as a series of steps, so an internal document should suffice for everyone to be on board with what needs to be done.
+In addition, it may be more useful to have more people in the team that are knowledgeable of how deployment works so the team does not depend on a single point of failure.
```

Comment:
```suggestion
In addition, it may be more useful to have more people in the team that are knowledgeable of how deployment works so the team does not have a single point of failure.
```